core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -511,6 +516,8 @@ object ConfigCommand extends Config {
       .ofType(classOf[String])
     val alterOpt = parser.accepts("alter", "Alter the configuration for the entity.")
     val describeOpt = parser.accepts("describe", "List configs for the given entity.")
+    val allOpt = parser.accepts("all", "List all configs (static and dynamic) for the given entity")
```

Comment:
Prefer not to refer to static and dynamic since they're specific to the broker configs (topics and quotas are all dynamic).

Reply:
Sounds good. I'll remove `static and dynamic` from the description.